packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-card.tsx

```diff
@@ -24,37 +19,47 @@ const JurisdictionCard = ({
     const verification_docs = is_synthetic
         ? card_values?.synthetic_verification_docs
         : card_values?.financial_verification_docs;
-    const is_card_flipped = card_flip_status[type_of_card];
+    const [is_card_flipped, setIsCardFlipped] = useState(false);
+    const is_card_selected = jurisdiction_selected_shortcode === type_of_card;
+
     const cardSelection = (cardType: string) => {
         setJurisdictionSelectedShortcode(jurisdiction_selected_shortcode === cardType ? '' : cardType);
     };
 
     const toggleCardFlip = (event: SyntheticEvent) => {
         event.stopPropagation();
-        flipCard(type_of_card);
+        setIsCardFlipped(!is_card_flipped);
     };
 
     return (
-        <div
-            className={classNames(card_classname, {
-                [`${card_classname}--selected selected-card`]: jurisdiction_selected_shortcode === type_of_card,
-            })}
-            onClick={disabled ? () => undefined : () => cardSelection(type_of_card)}
-        >
-            {!is_card_flipped ? (
+        <div className='cfd-card-perspective'>
+            <div
+                className={classNames(
+                    'cfd-card-container',
+                    {
+                        'cfd-card-flipped': is_card_flipped,
+                    },
+                    {
+                        'cfd-card-disabled': disabled,
+                    }
```

Comment:
shouldnt these classes added in one object?
```suggestion
                   className={classNames(
                    'cfd-card-container',
                    {
                        'cfd-card-flipped': is_card_flipped,
                         'cfd-card-disabled': disabled,
                    },
                   
```